org.eclipse.gef/src/org/eclipse/gef/editparts/AbstractGraphicalEditPart.java

```diff
@@ -498,8 +498,8 @@ protected IFigure getLayer(Object layer) {
 	 * @return the List of model source connections
 	 */
 	@SuppressWarnings("static-method")
-	protected List getModelSourceConnections() {
-		return Collections.EMPTY_LIST;
+	protected List<? extends Object> getModelSourceConnections() {
```

Comment:
I don't think it's necessary to explicitly extend Object.
```suggestion
	protected List<?> getModelSourceConnections() {
```

Reply:
Yes. Fixed.